add detailes about whether mpp takes effect (#5663)
pingcap/docs: tiflash/use-tiflash.md

set @@session.tidb_allow_mpp=0
```

> MPP mode supports these physical algorithms: Broadcast Hash Join, Shuffled Hash Join, and Shuffled Hash Aggregation. The optimizer automatically determines which algorithm to be used in a query. To check the specific query execution plan, you can execute the `EXPLAIN` statement. If ExchangeSender and ExchangeReceiver operators appear after you execute the `EXPLAIN` statement, then the MPP mode has taken effect.

Reviewer: ```suggestion
MPP mode supports these physical algorithms: Broadcast Hash Join, Shuffled Hash Join, and Shuffled Hash Aggregation. The optimizer automatically determines which algorithm to be used in a query. To check the specific query execution plan, you can execute the `EXPLAIN` statement. If the result of the `EXPLAIN` statement shows ExchangeSender and ExchangeReceiver operators, it indicates that the MPP mode has taken effect.
```
